src/mmw/js/src/core/chart.js

```diff
@@ -596,9 +596,45 @@ function renderCompareMultibarChart(chartEl, name, label, colors, stacked, yMax,
     }, onRenderComplete);
 }
 
+function renderBulletChart(chartEl, title, subtitle, data) {
+    var chart = nv.models.bulletChart(),
+        svg = makeSvg(chartEl),
+        datum = {
+            title: title,
+            subtitle: subtitle,
+            measures: [data],
+            color: '#48c5d1',
+        },
+        adjustment = 1;
+
+    // Calculate max range of chart
+    // by rounding up to the next place
+    while (data < 1) {
+        data *= 10;
+        adjustment -= 1;
+    }
+    while (data > 10) {
+        data /= 10;
+        adjustment += 1;
```

Comment:
`data` looks not to be used again after these adjustments (I checked in the REPL and the `measures: [data]` is passed by value.

Consider making a small utility function which would take a value of whatever magnitude and return its base 10 exponent & adding tests for < 1, > 10, and single digit values.